src/UI/Component/Dropzone/File/Factory.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace ILIAS\UI\Component\Dropzone\File;
+
+use ILIAS\UI\Component\Component;
+
+/**
+ * Interface Factory
+ *
+ * Describes a factory for file dropzones.
+ *
+ * @author  nmaerchy <nm@studer-raimann.ch>
+ * @author  Stefan Wanzenried <sw@studer-raimann.ch>
+ *
+ * @package ILIAS\UI\Component\Dropzone\File
+ */
+interface Factory {
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      The standard dropzone is used to drop files dragged from outside
+	 *      the browser window. The dropped files are presented to the user and
+	 *      can be uploaded to the server. Standard dropzones CAN be used in
+	 *      forms.
+	 *   composition: >
+	 *      Standard dropzones consist of a visible area where files can
+	 *      be dropped. They MUST contain a message explaining that it is possible to
+	 *      drop files inside. The dropped files are presented to the user along
+	 *      with some button to start the upload process.
```

Comment:
The upload button is optional, we are unsure if it is really needed ATM. If the standard dropzone is used in a (legacy) form, there is no upload button as the files are uploaded via the forms submit button. I will update the description accordingly.